scss/skin/_default.scss

```diff
@@ -67,6 +67,11 @@ video {
   @extend .oo-expand;
 }
 
+// fix anamorphic video issue on Firefox
+@include apply-to-firefox-only ('.oo-anamorphic video') {
+  object-fit: fill !important;
```

Comment:
correct aspect ratio maintained in fullscreen as well -

<img width="1440" alt="screen shot 2017-02-06 at 1 02 20 pm" src="https://cloud.githubusercontent.com/assets/3091495/22666232/9fb64708-ec6c-11e6-92e9-5761b668e7af.png">

<img width="1440" alt="screen shot 2017-02-06 at 1 02 25 pm" src="https://cloud.githubusercontent.com/assets/3091495/22666239/a7133d76-ec6c-11e6-8d03-b9fbee57674d.png">